Crashlytics/ProtoSupport/Protos/crashlytics.proto

```diff
@@ -0,0 +1,252 @@
+// Copyright 2020 Google
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto2";
+
+package com.google.crashlytics.proto;
```

Comment:
For shorter generated typenames, I'd just do `google_crashlytics` or something like that.  GDT's is 'gdt_cct'.